Makefile

```diff
@@ -29,19 +43,12 @@ test_core:
 
 # Runs `ffigen` for `mediapipe_text`
 generate_text:
-	cd packages/mediapipe-task-text && dart run ffigen --config=ffigen.yaml
-
-# Compiles the faked C artifacts for testing
-compile_fake_text:
-	# Builds standalone executable
-	cd packages/mediapipe-task-text/test/c && gcc fake_text_classifier.c -o fake_text_classifier
-	# Builds what Flutter needs
-	cd packages/mediapipe-task-text/test/c && gcc -static -c -fPIC *.c -o fake_text_classifier.o
-	cd packages/mediapipe-task-text/test/c && gcc -shared -o fake_text_classifier.dylib fake_text_classifier.o 
-
-# Runs `ffigen` for `mediapipe_text` and all text tests
-test_text: compile_fake_text test_text_only
+	cd packages/mediapipe-task-text && dart --enable-experiment=native-assets run ffigen --config=ffigen.yaml
 
 # Runs all text tests
-test_text_only:
+test_text:
 	cd packages/mediapipe-task-text && flutter test
+	cd packages/mediapipe-task-text/example && flutter test
+
+example_text:
+	cd packages/mediapipe-task-text/example && flutter run -d macos
```

Comment:
I tried to add a newline here so you would not have to, alas, Github is messing with me.